app/code/Magento/MediaGallery/Model/ResourceModel/SaveAssets.php

```diff
@@ -66,7 +58,18 @@ public function execute(array $assets): void
             try {
                 $connection->insertOnDuplicate(
                     $tableName,
-                    $this->filterData($this->objectProcessor->buildOutputDataArray($asset, AssetInterface::class))
+                    [
+                        'id' => $asset->getId(),
+                        'path' => $asset->getPath(),
+                        'title' => $asset->getTitle(),
+                        'source' => $asset->getSource(),
+                        'content_type' => $asset->getContentType(),
+                        'width' => $asset->getWidth(),
+                        'height' => $asset->getHeight(),
+                        'size' => $asset->getSize(),
+                        'created_at' => $asset->getCreatedAt(),
+                        'updated_at' => $asset->getUpdatedAt(),
```

Comment:
Update at and created at should not be saved to the database if their values are null